storage/waitlist/items.go

```diff
@@ -0,0 +1,242 @@
+package waitlist
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/iryonetwork/wwm/gen/waitlist/models"
+	"github.com/iryonetwork/wwm/storage/encrypted_bolt"
+	"github.com/iryonetwork/wwm/utils"
+)
+
+// ListItems returns all items in a waitlist
+func (s *storage) ListItems(waitlistID []byte) ([]*models.Item, error) {
+	var list []*models.Item
+
+	err := s.db.View(func(tx *bolt.Tx) error {
+		bCurrent := tx.Bucket(bucketCurrent).Bucket(waitlistID)
+		if bCurrent == nil {
+			return utils.NewError(utils.ErrNotFound, "waitlist not found")
+		}
+
+		var i byte
+		for i = 1; i <= priorityLevels; i++ {
```

Comment:
I considered it and thought this would be faster... Turns out it's a bit faster with larger lists, but not considerably:

```
BenchmarkListItems10-4        	  100000	     97846 ns/op
BenchmarkListItems20-4        	   50000	    169399 ns/op
BenchmarkListItems50-4        	   20000	    428438 ns/op
BenchmarkListItems100-4       	   10000	    852055 ns/op
BenchmarkListItemsSort10-4    	  100000	     86269 ns/op
BenchmarkListItemsSort20-4    	   50000	    178810 ns/op
BenchmarkListItemsSort50-4    	   10000	    510608 ns/op
BenchmarkListItemsSort100-4   	   10000	   1090585 ns/op
```